zerver/views/custom_profile_fields.py

```diff
@@ -78,6 +78,13 @@ def is_default_external_field(field_type: int, field_data: ProfileFieldData) ->
     return True
 
 
+# We allow updating only Name and Hint field of default external account field.
+def updating_name_and_hint_of_default_external_account(
+    existing_field_data: ProfileFieldData, requested_field_data: ProfileFieldData
+) -> bool:
+    return existing_field_data["subtype"] == requested_field_data["subtype"]
```

Comment:
This check incorrect, since `subtype` isn't the only thing that cap appear in these.